server/game/cards/01_SOR/events/MaximumFirePower.ts

```diff
@@ -0,0 +1,57 @@
+import { EventCard } from '../../../core/card/EventCard';
+import {
+    RelativePlayer,
+    Trait,
+    WildcardCardType,
+    WildcardLocation
+} from '../../../core/Constants';
+import AbilityHelper from '../../../AbilityHelper';
+import * as EnumHelpers from '../../../core/utils/EnumHelpers';
+export default class MaximumFirePower extends EventCard {
+    protected override getImplementationId () {
+        return {
+            id: '2758597010',
+            internalName: 'maximum-firepower',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.setEventAbility({
+            title: 'A friendly Imperial unit deals damage equal to its power to a unit.',
+            targetResolvers: {
+                firstImperial: {
+                    controller: RelativePlayer.Self,
+                    locationFilter: WildcardLocation.AnyArena,
+                    cardTypeFilter: WildcardCardType.Unit,
+                    cardCondition: (card) => card.hasSomeTrait(Trait.Imperial),
+                },
+                selectedTarget: {
```

Comment:
For clarity, can we call this `damageTarget` or something

Reply:
Yes sir!